This adds videos for Seasons 4-6 (#4177)
influxdata/docs-v2: content/resources/videos/Checks-&-Notification-in-Action.md

---
title: Checks & Notification in Action
description: >

>   In another video, Zoe Steinkamp discussed what checks and notifications in InfluxDB are. Here, she uses a real world example to demonstrate how and when to use the different types of checks and notifications available in InfluxDB. Intro to Checks and Notifications: https://youtu.be/-QdldB3RxMw

Reviewer: ```suggestion
Here, Zoe Steinkamp uses a real world example to demonstrate how and when to use the different types of checks and notifications available in InfluxDB. To learn about InfluxDB checks and notifications, see Zoe's video on [Intro to Checks and Notifications](https://youtu.be/-QdldB3RxMw).
```
